README.md

```diff
@@ -163,7 +161,7 @@ This plugin is in Beta testing stage. `gitlab-branch-source-0.0.7-beta` release
     
         6. Select `Install without restart`
 	
-4. Download `gitlab-branch-source-1.0.0.hpi` from [here](https://repo.jenkins-ci.org/releases/io/jenkins/plugins/gitlab-branch-source/1.0.0/gitlab-branch-source-1.0.0.hpi) and manually install.
+4. Download latest release (`gitlab-branch-source-1.2.1.hpi`) from [here](https://repo.jenkins-ci.org/releases/io/jenkins/plugins/gitlab-branch-source/1.2.1/gitlab-branch-source-1.2.1.hpi) and manually install.
```

Comment:
Might be better with a permalink and avoid mentioning the version here. That's what you have the badge for in the top :)
```suggestion
4. Download latest release from [here](http://updates.jenkins.io/latest/gitlab-branch-source.hpi) and manually install.
```